roles/convert2rhel/README.md

```diff
@@ -0,0 +1,41 @@
+theforeman.foreman.convert2rhel
+===========================
+
+This role creates a basic configuration for everything needed to register and convert CentOS clients to Red hat Enterprise Linux.
+
+First is upload of manifest & sync of RHEL 7 & RHEL 8 repositories, for more detail see [content_rhel Role](https://github.com/theforeman/foreman-ansible-modules/blob/develop/roles/content_rhel/README.md).
+
+The role creates then Convert2RHEL products & repositories (with synchronization), activation keys and host groups for each OS.
+
+If simple content access is disabled, subscriptions and repositories for RHEL activation keys must be added manually.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables) and [Content RHEL variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/roles/content_rhel/README.md)
+
+- `convert2rhel_lifecycle_env`: Lifecycle environment for activation keys, default: Library.
+- `convert2rhel_content_view`: Content view for activation keys, default: Default Organization View.
```

Comment:
```suggestion
- `foreman_convert2rhel_lifecycle_env`: Lifecycle environment for activation keys, default: Library.
- `foreman_convert2rhel_content_view`: Content view for activation keys, default: Default Organization View.
```

in `roles/convert2rhel/defaults/main.yml` there are more variables, should they also be documented?